examples/timeseries/md/timeseries_classification_from_scratch.md

```diff
@@ -177,15 +177,12 @@ model = make_model(input_shape=x_train.shape[1:])
 keras.utils.plot_model(model, show_shapes=True)
 ```
 
+<div class="k-default-codeblock">
+```
+You must install pydot (`pip install pydot`) for `plot_model` to work.
```

Comment:
Please install pydot and regenerate. This should show a model plot.

Reply:
Should be fixed now, also installed graphviz